### PR TITLE
Update CodeInsightSearchExample component styles

### LINKED ISSUE #5137: Debug code insights component style issue on Safari

### PR DESCRIPTION
**Pull request by katjuell**
Closes #5137 by setting `height` on the `CodeInsightSearchExample` component.

## To Test
- Go to the `code-insights` demo page.
- Check that both charts look okay on Safari and other browsers, and on mobile

### CODE REVIEW

**Comment by katjuell** [comment]
@vovakulikov just want to confirm that you can reproduce this problem on Safari at https://deploy-preview-5136--sourcegraph.netlify.app/code-insights 
Both @bretthayes and I were able to reproduce. I wonder if maybe you and I looked at staging on #5120 before the edge cache had updated. Looks like it's a height issue?

https://user-images.githubusercontent.com/43454804/153957319-124fc84c-b096-4ce2-927d-c34988837cce.mov

**Comment by vovakulikov** [comment]
@katjuell that it's weird that I didn't catch this in my previous PR about adding these charts but I can see this problem now. The good news is that I'm 95% sure what this problem is about. Try to add `position: absolute` to the element on the screenshot below. It should help with chart sizing.

Screenshot 2022-02-15 at 13 18 41

**Comment by katjuell** [comment]
@vovakulikov awesome — thanks for confirming and taking a look!

File: website/src/pages/code-insights.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+} from '../components/code-insights/mock-data'
+
+const styles = {
+    container: `
+        display: flex;
+        flex-wrap: wrap;
+        gap: 1rem;
+        margin: 1rem;
+    `,
+    insight: '',
```
Comment (bretthayes) [inline website/src/pages/code-insights.tsx:19]: Is this empty string intended?

**Review by bretthayes — APPROVED** [review]

**Pull request merged**